### PR TITLE
feat(backend): add send email action

### LINKED ISSUE #573: Enable the send email option when user don't receive any confirmation email
After the sign up process don't receive any confirmation email, the user must be able to request for a send email back again. Also is very important to tell him that the mail could be under the spam section.

- [x] Create content ES & EN @ldrojas 
- [x] Button placement design @murillojorge 
- [x] Create dialog box

Steps to test:
1. Go to sign up section
1. Sign up
1. Request for send email again

Own: @steph222 
Sup: @leisterfrancisco

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Resolves #573 

### What does this PR do?
Enable the send email again option if user have not received email verification yet

### How should this be tested?
- Make run
- Login without having verified the email
- Check email is send

### CODE REVIEW

**Review by leisterfrancisco — COMMENTED** [review]
@steph222
1. What if we add a Snackbar when the user click the RESEND button
2. After user click RESEND button, modals close, that's right, but if a click login again, it will show up resend email modal again, take a look at that

**Comment by Edgar097** [comment]
I am not receiving the verification email with my hotmail account, and i have this current problem when trying to log in 
image

**Review by Edgar097 — COMMENTED** [review]
look at the comments @leisterfrancisco @steph222

File: webapp/src/components/ResendComponent/ResendComponent.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      aria-labelledby="transition-modal-title"
+      aria-describedby="transition-modal-description"
+      fullScreen={!isDesktop}
+      closeAfterTransition
+      BackdropComponent={Backdrop}
+      BackdropProps={{
+        timeout: 500
+      }}
+    >
+
```
Comment (JustinCast) [inline webapp/src/components/ResendComponent/ResendComponent.js:49]: Delete this blank space

File: webapp/src/components/ResendComponent/ResendComponent.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    handlerSendEmail()
+    handlerOpen(false)
+  }
+
+  return (
+    <Dialog
+      open={open}
+      onClose={handlerOpen}
+      aria-labelledby="transition-modal-title"
+      aria-describedby="transition-modal-description"
```
Comment (JustinCast) [inline webapp/src/components/ResendComponent/ResendComponent.js:41]: These attributes seem to have their original placeholder

File: webapp/src/components/LoginModal/Login.js [hunk 3]
```diff
@@ -262,9 +329,15 @@ const LoginModal = ({ isNavBar, isSideBar }) => {
...
           </Box>
+          <ResendComponent
+            open={openVerify}
+            handlerOpen={handlerSetOpenVerify}
+            handlerSendEmail={handleSendEmail}
+          />
           <Box className={classes.credentialsBox}>
             <CredentialsRecovery />
           </Box>
+
```
Comment (JustinCast) [inline webapp/src/components/LoginModal/Login.js:340]: Delete this blank space

File: webapp/src/components/LoginModal/Login.js [hunk 4]
```diff
@@ -248,6 +314,7 @@ const LoginModal = ({ isNavBar, isSideBar }) => {
               >
                 {t('login.login')}
               </Button>
+
```
Comment (JustinCast) [inline webapp/src/components/LoginModal/Login.js:317]: Delete this blank space

File: webapp/src/components/LoginModal/Login.js [hunk 5]
```diff
@@ -107,6 +162,17 @@ const LoginModal = ({ isNavBar, isSideBar }) => {
...
 
+  useEffect(() => {
+    console.log('CHECK-EMAIL-VERIFIED-RESULT', checkEmailVerifiedResult)
+    if (checkEmailVerifiedResult && !checkEmailVerifiedResult.verified) {
+      verify()
+    }
+  }, [checkEmailVerifiedResult])
+
+  useEffect(() => {
+    console.log('SEND-EMAIL-RESULT', sendEmailResult)
```
Comment (JustinCast) [inline webapp/src/components/LoginModal/Login.js:173]: Remove this console.log

**Review by JustinCast — CHANGES_REQUESTED** [review]
@leisterfrancisco great job. Just added a few things we can improve

**Review by JustinCast — APPROVED** [review]
LGTM

**Pull request merged**